js/actions/tests/newSource.js

```diff
@@ -1,25 +1,28 @@
 "use strict";
 
-const { createStore, actions, queries } = require("../../util/test-head");
+const { createStore } = require("../../util/test-head");
+const { newSource } = require("../../actions");
+const { getSourceByActor } = require("../../queries");
+
 const store = createStore();
 var expect = require("chai").expect;
```

Comment:
Yea, that's a good idea. At this point, I'm numb to assertion libraries. I've given up hope that my muscle memory will know if it's camelCased or dot deliniated. Lets look around for a convention soon because it will be a pain to switch soon.